Bottom: 9eb9f23b515be2c5b37efb27948f0b3877954635
Top:    83ec14097473851b1f72bfa6e643d79b714718cc
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-27 16:29:46 -0700

Refresh of lldb-python-fix-another-fflush

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 070cfe27ffe..302901664ec 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -192,9 +192,9 @@ inline llvm::Error keyError() {
 #if PY_MAJOR_VERSION < 3
 // The python 2 API declares some arguments as char* that should
 // be const char *, but it doesn't actually modify them.
-char *py2_const_cast(const char *s) { return const_cast<char *>(s); }
+inline char *py2_const_cast(const char *s) { return const_cast<char *>(s); }
 #else
-const char *py2_const_cast(const char *s) { return s; }
+inline const char *py2_const_cast(const char *s) { return s; }
 #endif
 
 enum class PyInitialValue { Invalid, Empty };
